libs/langchain/langchain/document_loaders/mintbase.py

```diff
@@ -0,0 +1,6 @@
+from langchain_community.document_loaders.mintbase import (
```

Comment:
Could you remove this change -- we're breaking apart community and langchain, so that community imports have to be imported from langchain_community only